src/nvim/testdir/test_quickfix.vim

```diff
@@ -1012,9 +1012,9 @@ endfunc
 func Test_efm_dirstack()
   " Create the directory stack and files
   call mkdir('dir1')
-  call mkdir('dir1/a')
-  call mkdir('dir1/a/b')
-  call mkdir('dir1/c')
+  call mkdir('dir1/a', 'p')
+  call mkdir('dir1/a/b', 'p')
+  call mkdir('dir1/c', 'p')
```

Comment:
Not seeing the value of the `'p'` flag when the previous `mkdir()` aren't collapsed to 1 line.
This isn't in Vim but it can be upstreamed.